### PR TITLE
STOR-956: Add artifacts to generate bundle

### PR DESCRIPTION
**Pull request by bertinatto**
CC @openshift/storage

### CODE REVIEW

File: config/manifests/stable/gcp-filestore-csi-driver-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+    categories: Storage
+    "operatorframework.io/suggested-namespace": openshift-cluster-csi-drivers
+    capabilities: Full Lifecycle
+    containerImage: quay.io/openshift/origin-gcp-filestore-csi-driver-operator:latest
+    support: Red Hat
+    operators.openshift.io/infrastructure-features: '["csi"]'
+    repository: https://github.com/openshift/gcp-filestore-csi-driver-operator
+    createdAt: "2021-07-14T00:00:00Z"
+    description: Install and configure GCP Filestore CSI driver.
+    olm.skipRange: ">=4.11.0-0 <4.12.0"
```
Comment (bertinatto) [inline config/manifests/stable/gcp-filestore-csi-driver-operator.clusterserviceversion.yaml:16]: I'm not sure about this one. @gnufied can you take a look?

Comment (gnufied) [reply config/manifests/stable/gcp-filestore-csi-driver-operator.clusterserviceversion.yaml:16]: skipRange looks fine.

File: config/bundle.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM scratch
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=gcp-filestore-csi-driver-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
```
[the remark this answers is not in the document]
Comment (bertinatto) [reply config/bundle.Dockerfile]: Because this is Tech Preview, I renamed everything to `preview`.

**Comment by gnufied** [comment]
nit: can you add a README that documents how to use the hack script?

**Pull request merged**